pkg/kubelet/network/dns/dns_test.go

```diff
@@ -92,7 +92,10 @@ func TestParseResolvConf(t *testing.T) {
 		{"#comment\nnameserver 1.2.3.4\n#comment\nsearch foo\ncomment", []string{"1.2.3.4"}, []string{"foo"}, []string{}, false},
 		{"options ", []string{}, []string{}, []string{}, false},
 		{"options ndots:5 attempts:2", []string{}, []string{}, []string{"ndots:5", "attempts:2"}, false},
+		{"options ndots:1 ndots:5 attempts:2", []string{}, []string{}, []string{"ndots:5", "attempts:2"}, false},
```

Comment:
can you do some test cases with options that doesn't have the `:` ?
like `edns0` per example

Reply:
Added a test case.
P.S. I tested that locally before.